### PR TITLE
Update enums.md

### PR DESCRIPTION
**Pull request by wanghengxn**
enum name is mandatory

### CODE REVIEW

**Review by mshima — APPROVED** [review]

File: pages/jdl/enums.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ sitemap:
 Enumeration declaration is done as follows:
 
 ```
-enum [<enum name>] {
-  <ENUM KEY> ([<enum value>])
+enum <enum name> {
```
Comment (MathieuAA) [inline pages/jdl/enums.md:27]: thanks

File: pages/jdl/enums.md [hunk 2]
```diff
@@ -24,8 +24,8 @@ sitemap:
...
+  <ENUM KEY> [(<enum value>)]
```
Comment (MathieuAA) [inline pages/jdl/enums.md:28]: I don't really see the point here

Comment (mshima) [reply pages/jdl/enums.md:28]: `([<enum value>])` implies that this is valid:
```
enum Foo {
  Bar ()
}
```
And this is invalid
```
enum Foo {
  Bar
}
```

Comment (MathieuAA) [reply pages/jdl/enums.md:28]: Ah very good point! Your eyes are better than mine

**Review by MathieuAA — APPROVED** [review]

**Pull request merged**